package.json

```diff
@@ -16,23 +16,24 @@
     "cross-env": "^5.1.4",
     "eslint": "^4.19.1",
     "eslint-config-google": "^0.9.1",
-    "eslint-config-prettier": "^2.9.0",
+    "eslint-config-prettier": "^3.0.1",
     "eslint-plugin-flow-vars": "^0.5.0",
     "eslint-plugin-flowtype": "^2.46.1",
     "eslint-plugin-import": "^2.9.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
-    "eslint-plugin-prettier": "^2.6.0",
+    "eslint-plugin-prettier": "^2.6.2",
     "eslint-plugin-react": "^7.7.0",
     "flow-bin": "^0.42.0",
     "fs-extra": "^7.0.0",
     "glob": "^7.1.1",
+    "husky": "0.14.3",
     "jest": "^23.5.0",
     "jest-cli": "^23.5.0",
     "lerna": "^3.3.0",
+    "lint-staged": "7.3.0",
     "npm-run-all": "4.1.2",
     "plop": "^1.8.1",
-    "prettier": "^1.12",
-    "prettier-eslint-cli": "4.7.1",
```

Comment:
no need to for prettier-eslint, we can simply use eslint with the config/plugin installed.. more flexibility.